practice/completer/site.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * Completer object contains completions.
+ *
+ * completions will have keys as the completion and values as weights.
+ */
+function Completer() {
+  this.completions = {};
```

Comment:
Since you're storing ```completions``` as an object rather than an array, it might be more clear to use our dict naming conventions and call it something like ```completionsToCount```. This also makes some of your method functions in ```modifyCompleter``` more clear as the numbers which are stored as values are explained in the variable name rather than looking like magic numbers.